### PR TITLE
Kibana versions

### PR DESCRIPTION
**Pull request by jasontedor**
This pull request adds versioned Kibana formulae for the 4.1.x and 4.2.x lines of Kibana.

### CODE REVIEW

**Comment by jasontedor** [comment]
Blocked by #47500.

**Comment by jasontedor** [comment]
The 4.2.x build is effectively the same as the main Kibana formula for the latest stable, but the 4.1.x build does have some modifications.

File: kibana41.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  desc "Analytics and search dashboard for Elasticsearch"
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://github.com/elastic/kibana.git", :tag => "v4.1.4", :revision => "3df681b3f205b2b75e11ddf68c0985f80460d7bc"
+  head "https://github.com/elastic/kibana.git"
+
+  bottle do
+    sha256 "a324b8d38a6f488aedbbaf58410e247e7319c502e259c889e68a35a829a2e757" => :el_capitan
+    sha256 "29b5cc931a3bcfbd6df658b1f7a160245a4afea9fb8818689a99420dfb5c36b8" => :yosemite
+    sha256 "9036742597f5ef7cbea88f4e04f9d3a0227a3effad6ce9547142b31efdc67cd1" => :mavericks
+  end
```
Comment (bfontaine) [inline kibana41.rb]: Please remove the bottle blocks; they’re only relevant for the `kibana` core formula.

Comment (jasontedor) [inline kibana41.rb]: Removed in 122982dc655d5de3fd4a65814156ca8e99c22b35 and 46500e18cfc45eff2ae0326e4069d6f99acaaf20.

**Comment by mikemcquaid** [comment]
Have triggered a rebuild for this.

**Comment by jasontedor** [comment]
@mikemcquaid I pushed a pair of commits to address the failing style checks. (I'm pushing pairs so it's easier to reorder and squash the commits for each formula on a rebase.)

**Comment by mikemcquaid** [comment]
Thanks again @jasontedor and particularly for making it easier to squash!

**Pull request closed without merging**